src/app/app.routes.ts

```diff
@@ -26,6 +26,7 @@ export interface RouteData {
   isPublic?: boolean;
 
   dialogToken?: DialogComponentToken;
```

Comment:
I believe if we fully switch over to the CDK Dialog system, this property should be unused. We should aim to get this property in a removable (and then removed) state.